describealign.py

```diff
@@ -1110,33 +1110,55 @@ def combine_gui(video_files, audio_files, config_path):
       break
   combine_window.close()
 
-def migrate_config(old_path: Path, new_path: Path) -> None:
+def migrate_config(old_paths: list[Path], new_path: Path) -> bool:
   """
   Migrate configuration from old location.
   
-  Only runs if the old_path exists but new_path does not
+  Only runs if one of old_paths exists but new_path does not
+  
+  :param old_paths: A list of previous config file locations, in chronological order oldest to newest
+  :param new_path: The new config location
+  :returns: True if the config was migrated, else False
   """
-  if new_path.exists() or not old_path.exists():
-    return
+  if new_path.exists():
+    return False
   
-  old_data = old_path.read_text(encoding='utf-8')
-  new_path.write_text(old_data, encoding='utf-8')
-  print(f"Configuration migrated to {new_path}")
-  try:
-    old_path.unlink()
-  except OSError as exc:
-    print("Failed to remove old config:", *traceback.format_exception_only(exc))
-  else:
-    print("Successfully removed old config file.")
+  # Ensure the current path isn't in the old list
+  if new_path in old_paths:
+    old_paths.remove(new_path)
```

Comment:
Should probably throw an error, can't imagine that'd ever be intentional

Reply:
On macos, this is the case as platformdirs doesn't do the nested dir names there. I prefer to handle this generically as this can happen for a number of reasons but we know how to handle it